static/styles/portico/portico.scss

```diff
@@ -229,8 +237,7 @@ html {
     cursor: pointer;
 }
 
-.help .sidebar li a,
-.help .sidebar h1 a {
+.help .sidebar li a {
     color: inherit;
     display: block;
 }
```

Comment:
The whole structure of the file could have been written as the following for better redability,
```
.help {
    .sidebar {
        h1 {
           ...
        }
        li a {
            ...
        }
    }
}
```
Instead you are using separate tags.